### PR TITLE
[TIMOB-17675] iOS: Add support for SplitWindow on iPhone

### PR DESCRIPTION
**Pull request by vishalduggal**
Jira Ticket [TIMOB-17675](https://jira.appcelerator.org/browse/TIMOB-17675)

### CODE REVIEW

File: apidoc/Titanium/UI/iOS/SplitWindow.yml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    type: Boolean
+    default: false
+    
+  - name: masterIsOverlayed
+    summary: Determines whether to show the master view is overlayed in portrait orientation. 
+    type: Boolean
+    default: false
+    
+  - name: portraitSplit
+    summary: Determines the width of the `masterView` in portrait mode.
```
Comment (cheekiatng) [inline apidoc/Titanium/UI/iOS/SplitWindow.yml]: should mention `from 0.25 to 0.5`

File: apidoc/Titanium/UI/iOS/SplitWindow.yml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    type: Number
+    default: Undefined
+    
+  - name: landscapeSplit
+    summary: Determines the width of the `masterView` in landscape mode.
```
Comment (cheekiatng) [inline apidoc/Titanium/UI/iOS/SplitWindow.yml]: same here. should mention `from 0.25 to 0.5`

**Comment by vishalduggal** [comment]
Done. Ready for review.

**Comment by cheekiatng** [comment]
CR and FT passed. Merging.

**Comment by hansemannn** [comment]
This PR does **not** use the native `UISplitViewController` like proposed in the linked ticket. Does that have a reason?

**Pull request merged**